### PR TITLE
Decal cross fade support

### PR DESCRIPTION
**Pull request by FrancescoC-unity**
Fix for : https://fogbugz.unity3d.com/f/cases/1259395/

The issue with decals looking like they double applied is ultimately due to the fact that they did not support cross fading, hence they applied twice (once per LOD). 

**What have I tested**: the repro case (**NOTE** you need to enable cross fade on the shader graph for the decal in the repro)  and similar thing with a non shader graph decal material.

### CODE REVIEW

**Review by alelievr — APPROVED** [review]

File: com.unity.render-pipelines.high-definition/Runtime/Material/Decal/Decal.shader [hunk 1]
```diff
@@ -78,6 +78,8 @@ Shader "HDRP/Decal"
     #pragma shader_feature_local _MATERIAL_AFFECTS_MASKMAP
 
     #pragma multi_compile_instancing
+    // enable dithering LOD crossfade
+    #pragma multi_compile _ LOD_FADE_CROSSFADE
```
Comment (sebastienlagarde) [inline com.unity.render-pipelines.high-definition/Runtime/Material/Decal/Decal.shader:82]: If I understand the issue correctly this should only be apply with decal mesh right?
or it works with decal projector? if it apply only to decal mesh, then the 
#pragma multi_compile _ LOD_FADE_CROSSFADE
must be move in DecalMeshForwardEmissive and DBufferMesh pass (same for shader graph)

Comment (FrancescoC-unity) [reply com.unity.render-pipelines.high-definition/Runtime/Material/Decal/Decal.shader]: Done

**Comment by sebastienlagarde** [comment]
merging for 10.1 package

**Comment by TomasKiniulis** [comment]
Repro case testing status looks sufficient. No additional qa testing required

**Pull request merged**